src/bodies.cpp

```diff
@@ -1071,8 +1071,8 @@ void bodies::ConvexMesh::computeBoundingSphere(BoundingSphere& sphere) const
 
 void bodies::ConvexMesh::computeBoundingCylinder(BoundingCylinder& cylinder) const
 {
-  cylinder.length = mesh_data_ ? mesh_data_->bounding_cylinder_.length : 0.0;
-  cylinder.radius = mesh_data_ ? mesh_data_->bounding_cylinder_.radius : 0.0;
+  cylinder.length = mesh_data_ ? mesh_data_->bounding_cylinder_.length * scale_ + 2 * padding_ / sqrt(3) : 0.0;
+  cylinder.radius = mesh_data_ ? mesh_data_->bounding_cylinder_.radius * scale_ + padding_ / sqrt(3) : 0.0;
```

Comment:
I did not look through the whole patch here, so just one comment here:

Assuming this patch is correct, I believe it is justified to add a comment explaining where the `sqrt(3)` comes from.

Reply:
For meshes without padding, the bounding cylinder will be tight.